docs/sources/reference/commandline/cli.md

```diff
@@ -1720,6 +1720,9 @@ You can remove an image using its short or long ID, its tag, or its digest. If
 an image has one or more tag or digest reference, you must remove all of them
 before the image is removed.
 
+If you use the `-f` flag and specify the image's short or long ID, then this
+command untags all tags matching the ID and removes the image itself.
```

Comment:
I found this very confusing, in part because the example code that follows it is not an example of `-f` usage. As a result, I'm still not quite sure what `-f` does, and I don't know what the example is supposed to be showing me. Below, we have an example of the flag being used to remove images, but this sentence seems to be saying that `-f` is used to remove tags.

I *think* (but am not sure, since I can't actually test the command atm), that this sentence belongs after the one at ln. 1748. Further, we can re-cast the sentence to make it clearer: "The `-f` flag is used to untag and remove all images that match the specified ID."